### PR TITLE
add: list components

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: components/list/list.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                        </li>
+                        <li>
+                            <div class="card-title">Non Fiction</div>
+                            <div class="card-content">Nonfiction is any document or media content that intends, in good
+                                faith, to present only truth and accuracy regarding information, events, or people.
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+                <div class="code-box">
```
Comment (VishalPatil18) [inline components/list/list.html:151]: A suggestion here is that there's an unnecessary scroll. You could adjust the height of the container accordingly which avoids scrolling.
Screenshot 2022-02-01 at 16 20 34

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
Just one suggestion was added. Apart from that, the lists are quite good. The color combination is also beautiful! Great work 👍

File: components/list/list.html (hunk 1 above)
Comment (vermasavita) [reply components/list/list.html:151]: Let's say if I fix the height. then for small code that will be a big container.

Comment (VishalPatil18) [reply components/list/list.html:151]: When you embed the code, in the iframe you have to change the height for each one individually.
This is the height you should change
 
Screenshot 2022-02-02 at 09 23 34

**Review by ruchiragoswami — APPROVED** [review]
Looks good Savita :)

**Review by VishalPatil18 — APPROVED** [review]

**Pull request merged**